massa-models/src/active_block.rs

```diff
@@ -37,4 +37,13 @@ pub struct ActiveBlock {
     pub roll_updates: RollUpdates,
     /// list of (period, address, did_create) for all block/endorsement creation events
     pub production_events: Vec<(u64, Address, bool)>,
+    /// Slot of the block.
+    pub slot: Slot,
+}
+
+impl ActiveBlock {
```

Comment:
Because it appears like that method is only used in the context of an active block(and with the changes introduced in this PR, the block itself is not stored in the active block anymore).